pkg/api/common/helper.go

```diff
@@ -14,6 +14,44 @@ import (
 	validator "gopkg.in/go-playground/validator.v9"
 )
 
+var (
+	/* If a new GPU sku becomes available, add a key to this map, but only if you have a confirmation
+	   that we have an agreement with NVIDIA for this specific gpu.
+	*/
+	NvidiaEnabledSKUs = map[string]bool{
```

Comment:
For future reference - we don't want to diverge from aks-e code when absolutely necessary.